### PR TITLE
A11y Support in CI

### PR DESCRIPTION
**Pull request by seanforyou23**
This PR adds support for attaching custom a11y reporting to our continuous integration. It also improves the a11y audit reporter by ironing out some small bugs and maximizing what can be tested depending on the environment.

This is an incremental step toward solving https://github.com/patternfly/patternfly-next/issues/522

This addresses exposing the results for contributors, but not yet storing or publishing the results in any persistent manner. Publishing the results has quite a number of considerations on its own, so I thought it would be good to include that as a separate PR.

So, if a user posts a PR that introduces more a11y violations than we currently have, they'll see a red x for continuous-integration/travis-ci/pr status reporter and can select the Details link for it which will take them to the relevant build log, where they can find a message that looks like below.

screen shot 2018-08-17 at 10 00 45 am

Information related to the errors can be found by simply scrolling up and viewing the logs produced by the audit reporter.

### CODE REVIEW

File: src/site/layouts/demo.js [hunk 1]
```diff
@@ -7,7 +7,6 @@ export default ({ children }) => (
   <main>
     <Helmet>
       <title>PF Next</title>
-      <html lang="en-US" />
```
Comment (dmiller9911) [inline src/site/layouts/demo.js]: why was this removed?

Comment (seanforyou23) [reply src/site/layouts/demo.js]: Just to illustrate, in this PR, what it looks like when the build fails. I'm going to "fix" it in just a few, but wanted to get some eyes on the unhappy path first since it's what new contributors will be seeing for the first time.

Comment (dmiller9911) [reply src/site/layouts/demo.js]: HAHA. I was like "what are you doin! It is going to fail!" Good idea for the illustration.
